robot_ipsa:
  l0: .1
  l1: .6
  l2: .4
  l3: .2
  l4: .05
  alpha: rad(0)
  lim1_lower: 0
  lim1_upper: rad(pi)
  lim2_lower: rad(-pi)
  lim2_upper: 0
  lim3_lower: rad(-pi)
  lim3_upper: rad(pi/2)
